sparse_strips/vello_cpu/src/fine.rs

```diff
@@ -0,0 +1,176 @@
+// Copyright 2025 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Fine rasterization runs the commands in each wide tile to determine the final RGBA value
+//! of each pixel and pack it into the pixmap.
+
+use crate::util::ColorExt;
+use vello_common::coarse::{Cmd, WIDE_TILE_WIDTH};
+use vello_common::paint::Paint;
+use vello_common::strip::STRIP_HEIGHT;
+
+pub(crate) const COLOR_COMPONENTS: usize = 4;
+pub(crate) const TOTAL_STRIP_HEIGHT: usize = STRIP_HEIGHT * COLOR_COMPONENTS;
```

Comment:
I don't love this name, it's not a height. I'm not sure what a less clunky name would be, maybe `STRIP_HEIGHT_COMPONENTS`?